neurom/features/tests/test_get_features.py

```diff
@@ -118,14 +118,14 @@ def test_section_tortuosity_pop():
                      4.657,
                      440.408,
                      1.342),
-                    rtol=1e-3)
+                    rtol=1e-2)
```

Comment:
The error is slightly too much I'd say, need to investigate.

Reply:
It happened because `Neuron.h5` is updated in this PR, and it has one section less than the previous.